.github/workflows/audit.yml

```diff
@@ -8,17 +8,26 @@ concurrency:
 
 jobs:
   build:
-    name: NPM audit
+    name: PNPM audit
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # https://github.com/actions/checkout/releases/tag/v4.1.1
 
+      - uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4.1.0
+        name: Install pnpm
+        with:
```

Comment:
Is it possible to read that value from file? As we did with npm and `.npmrc` file